lib/src/components/Banner.tsx

```diff
@@ -9,13 +9,71 @@ type TBanner = {
 	image?: string,
 	primaryButton?: ReactElement,
 	secondaryButton?: ReactElement,
-	children: ReactElement | ReactElement[],
+	children?: ReactElement | ReactElement[],
 	canClose?: boolean,
 	onClose?: () => void,
 	onClick?: () => void,
-	maxHeight?: string,
 	variant?: 'default' | 'image' | 'split' | 'background'
 }
+type TDefaultVariant = {
+	title?: string,
+	primaryButton?: ReactElement,
+	secondaryButton?: ReactElement,
+	children?: ReactElement | ReactElement[],
+	variant?: 'default' | 'image' | 'split' | 'background'
+}
+
+function	SplitVariant({image}: {image: string}) {
+	return (
+		<div className={'w-full md:w-1/2 relative'}>
+			<div
+				className={'flex md:hidden overflow-hidden relative -mx-1 -mt-1 w-full h-full rounded-xl border-x-2 border-b-2 border-primary max-h-48'}
+				style={{width: 'calc(100% + 8px)'}}>
+				<div className={'rounded-lg image-align-middle'}>
+					<img src={image} className={'w-full h-full object-cover'} loading={'eager'} />
+				</div>
+			</div>
+			<div
+				className={'hidden md:flex overflow-hidden -my-1 -ml-1 w-full h-full rounded-xl border-y-2 border-l-2 border-primary absolute'}
+				style={{height: 'calc(100% + 8px)', width: 'calc(100% + 8px)'}}>
+				<div
+					style={{minWidth: 'calc(100% + 8px)'}}
+					className={'rounded-lg image-align-middle w-full h-full -ml-1'}>
+					<img src={image} className={'w-full h-full object-cover'} loading={'eager'} />
+				</div>
+			</div>
+		</div>
+	)
+}
+
+function	BackgroundVariant({image}: {image: string}) {
+	return (
+		<div className={'absolute inset-0 w-full h-full -ml-1 -z-10 img-gradient'} style={{minWidth: 'calc(100% + 8px)'}}>
+			<img src={image} className={'relative object-cover w-full h-full'} loading={'eager'} />
+		</div>
+	)
+}
+
+function	DefaultVariant({variant, title, children, primaryButton, secondaryButton}: TDefaultVariant) {
+	return (
+		<div className={`p-4 md:p-6 flex-col ${variant === 'split' && 'w-full md:w-1/2'}`}>
+			<h4 className={'mb-4 md:mb-6 text-inherit'}>{title}</h4>
+			<div className={'mb-4 md:mb-6 text-inherit'}>{children}</div>
+			<div className={'flex flex-col md:flex-row space-y-4 md:space-y-0 space-x-0 md:space-x-4'}>
+				{primaryButton}
+				{secondaryButton}
+			</div>
```

Comment:
Instead of having spaces inside, we put the spaces on the wrapper with `space-x-4`. This will not add spaces if there is only one button or none. The space-y etc. are used for responsivness